### PR TITLE
WIP F added isactive flag

### PR DESCRIPTION
**Pull request by LarsTepper**

### CODE REVIEW

**Comment by LarsTepper** [comment]
Only ui needed to activate an deactivate flows! very usefull for debugging

File: src/Simplic.Flow.Service/FlowService.cs [hunk 1]
```diff
@@ -180,7 +180,7 @@ private IList<Flow> CreateFlowsFromConfiguration()
             flowLogService.Info($"> Creating flows from configurations...");
 
             var list = new List<Flow>();
-            foreach (var flowConfiguration in flowConfigurations)
+            foreach (var flowConfiguration in flowConfigurations.Where(f=>f.IsActive))
```
Comment (simplicbe) [inline src/Simplic.Flow.Service/FlowService.cs]: Strg + K + D

**Review by simplicbe — CHANGES_REQUESTED** [review]
Reset line ending

**Review by yvz5 — CHANGES_REQUESTED** [review]
`src/Simplic.FlowInstance.Service/FlowInstanceService.cs`
why is this file changed ? probably line ending style changed.

**Comment by yvz5** [comment]
merged into #50

**Pull request closed without merging**